Lib/fontbakery/commands/check_adobefonts.py

```diff
@@ -2,28 +2,22 @@
 import sys
 
 from functools import partial
-from fontbakery.profiles.adobefonts import profile
+from fontbakery.profiles.adobefonts import profile, BLACKLISTS
```

Comment:
Please do not use terms like `BLACKLISTS` or `WHITELISTS`. Per Google's Respectful Language Policy, please use `BLOCKLISTS` and `ALLOWLISTS` instead.

Reply:
> What _is_ being blocked by the blocklists? I can't see anything. Do we even need that code at all?

Yeah, I think we can remove them. @graphicore, is there a need for these? Or can we delete them?